components/services/services.html

```diff
@@ -0,0 +1,39 @@
+<section class="services">
+  <h1 class="services__header">
+    {{ component.title }}
+  </h1>
+  <div class="boxes">
+    <div class="box">
+      <h4 class="box__title">
+        {{ component.box1.title }}
+      </h4>
+      <p class="box__desc">
+        {{ component.box1.desc }}
+      </p>
+    </div>
+    <div class="box">
+      <h4 class="box__title">
+        {{ component.box2.title }}
+      </h4>
+      <p class="box__desc">
+        {{ component.box2.desc }}
+      </p>
+    </div>
+    <div class="box">
+      <h4 class="box__title">
+        {{ component.box3.title }}
+      </h4>
+      <p class="box__desc">
+        {{ component.box3.desc }}
+      </p>
+    </div>
+    <div class="box">
+      <h4 class="box__title">
+        {{ component.box4.title }}
+      </h4>
+      <p class="box__desc">
+        {{ component.box4.desc }}
```

Comment:
Whenever possible use full names, keeps things readable and consistent, no risk of two devs using slightly different abbreviations to mean the same thing.